common-lib/src/main/scala/com/gu/mediaservice/lib/cleanup/MetadataCleaner.scala

```diff
@@ -18,7 +18,6 @@ class MetadataCleaners(resources: ImageProcessorResources)
     BylineCreditReorganise,
     UseCanonicalGuardianCredit,
     ExtractGuardianCreditFromByline,
-    AttributeCreditFromByline.fromPublications(resources.commonConfiguration.usageRightsConfig.allPhotographers),
```

Comment:
am I right in thinking that this is being removed as a pass-by refactor because it wasn't being used?

Reply:
ish - it was being run, but it's a duplicate of this supplier processor https://github.com/guardian/grid/blob/main/common-lib/src/main/scala/com/gu/mediaservice/lib/cleanup/SupplierProcessors.scala#L26 (the difference between a supplier processor and a cleaner isn't particularly well defined, but I'd argue this is more of a "processing" than a "cleaning")